fiaas_skipper/web/templates/deploy.html

```diff
@@ -1,5 +1,5 @@
 {%- extends "base.html" %}
-
+{% block body_attribs %} class="deploy" onLoad="onDeployPageLoad"{% endblock %}
```

Comment:
It is tempting to suggest that the URL to post to should be given as a parameter here, generated using the `url_for` method. That way it will still work even if we should happen to change the URL later. It's a minor thing, so feel free to ignore.